src/macros.rs

```diff
@@ -11,7 +11,7 @@ macro_rules! wgpu_profiler {
     ($label:expr, $profiler:expr, $encoder_or_pass:expr, $device:expr, $code:expr) => {{
         $profiler.begin_scope($label, $encoder_or_pass, $device);
         let ret = $code;
-        $profiler.end_scope($encoder_or_pass);
+        $profiler.end_scope($encoder_or_pass).unwrap();
```

Comment:
Hm, true. I'm trying to figure if we can instead use the `Scope` classes here, but that's ofc not easily possible. Might be also worth considering to deprecate the macro. If somebody wants it, they can very very easily do it themselves and add a feature flag to make it no-op.
A `disable-gpuprofiler` feature is something I wanted for this crate but it gets confusing in here quickly because it's unclear how far that should go (never create anything in the first place or only affect the macro or the `Scope` classes)

Reply:
I'm don't know how perfectly panic free you want to make this but I'm happy with either approach of keeping it as is (maybe documenting this case) or letting the user implement this.

My inclination for a feature flag would be to at least take it all the way to disabling the `start_scope`/`end_scope` methods.